### PR TITLE
Nest gamelab/ and spritelab/ under p5lab/ directory

### PR DESCRIPTION
**Pull request by ajpal**
See [doc](https://docs.google.com/document/d/1xJb79MH0Ga1pypFNyttVeNfmHm-tBWNyjGngf2dRfWs/) for more details.
This PR sets up the directory structure we want and changes a bunch of imports.
This does not create the base class that Sprite Lab and Game Lab will inherit from. I'll do that in the next PR, but wanted to get the big diff of changing imports out of the way first.

Part 2: #29918 
Part 3: #29925

### CODE REVIEW

File: apps/src/p5lab/gamelab/sharedGamelabBlocks.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
+/* eslint-disable */
+
+// This is a generated file and SHOULD NOT BE EDITED MANUALLY!!
```
Comment (jmkulwik) [inline apps/src/p5lab/gamelab/sharedGamelabBlocks.js]: Should this be added to the gitignore?

File: apps/test/unit/gamelab/spritelab/spriteUtilsTest.js [hunk 2]
```diff
@@ -2,8 +2,8 @@
 import {expect} from '../../../util/reconfiguredChai';
 import {stub} from 'sinon';
 import createGameLabP5 from '../../../util/gamelab/TestableGameLabP5';
-import * as spriteUtils from '@cdo/apps/gamelab/spritelab/spriteUtils';
-import {commands as spriteCommands} from '@cdo/apps/gamelab/spritelab/spriteCommands';
+import * as spriteUtils from '@cdo/apps/p5lab/gamelab/spritelab/spriteUtils';
+import {commands as spriteCommands} from '@cdo/apps/p5lab/gamelab/spritelab/spriteCommands';
```
Comment (jmkulwik) [inline apps/test/unit/gamelab/spritelab/spriteUtilsTest.js:6]: Should these be moved to p5lab/spritelab?

Comment (jmkulwik) [reply apps/test/unit/gamelab/spritelab/spriteUtilsTest.js:6]: Nevermind: This happens as part of https://github.com/code-dot-org/code-dot-org/pull/29925

File: apps/src/p5lab/gamelab/sharedGamelabBlocks.js (hunk 1 above)
Comment (ajpal) [reply apps/src/p5lab/gamelab/sharedGamelabBlocks.js]: Yes, good catch, thanks!

**Review by jmkulwik — APPROVED** [review]

**Pull request merged**